test/format/format.sh

```diff
@@ -215,6 +215,8 @@ report_failure()
 	echo "$name: job in $dir failed"
 	echo "$name: $dir log:"
 	sed 's/^/    > /' < $log
+	echo "$name: $dir/CONFIG:"
+	sed 's/^/    > /' < $dir/CONFIG
```

Comment:
I think just pasting the CONFIG into the log is fine. We don't need to add in `  >` because that means you have to hand edit that out.